components/brave_wallet_ui/page/screens/swap/components/swap/quote-info/quote-info.tsx

```diff
@@ -469,57 +495,33 @@ export const QuoteInfo = (props: Props) => {
                   >
                     {getLocale('braveWalletRoute')}
                   </Text>
-                  <Row width='unset'>
-                    <LPIcon icon={LPMetadata[selectedQuoteOption.provider]} />
+                  <Row
+                    width='unset'
+                    gap='8px'
+                  >
+                    {firstRoutesIcon ? (
+                      <LiquidityProviderIcon
+                        icon={firstRoutesIcon}
+                        size='16px'
+                      />
+                    ) : (
+                      <BankIcon size='16px' />
+                    )}
                     <Text
                       textSize='12px'
                       isBold={true}
                       textColor='primary'
                     >
-                      {selectedQuoteOption.provider}
+                      {firstRoutesName}{' '}
+                      {additionalRoutesLength !== 0
+                        ? `+ ${additionalRoutesLength} routes`
+                        : ''}
```

Comment:
@Douglashdaniel Paraswap + 2 routes should instead be called Paraswap + 2 steps.

Nice to have: we may have a single additional step, hence it'll be Paraswap + 1 step.

Reply:
`step` should use i18n

Reply:
Done and localized!